scripts/install_operator.sh

```diff
@@ -38,6 +35,3 @@ kubectl apply --namespace "${NAMESPACE}" -f ${SPECS_DIR}/spark-driver-rbac.yaml
 # TODO: make the spark-application-metrics-service.yaml manageable by the kudo-spark-operator once the issue https://github.com/kudobuilder/kudo/issues/916 will be fixed.
 kubectl apply --namespace "${NAMESPACE}" -f ${SPECS_DIR}/spark-application-metrics-service.yaml
 
-# Create ServiceMonitor (see prometheus-operator docs) for Spark.
-# TODO: make the service-monitor.yaml manageable by the kudo-spark-operator once the issue https://github.com/kudobuilder/kudo/issues/913 will be fixed.
-kubectl apply --namespace "${NAMESPACE}" -f ${SPECS_DIR}/spark-service-monitor.yaml
```

Comment:
can you please remove that from scripts in `scale-tests`  too

Reply:
Removed